docs/source/docker/getting_started/selection.rst

```diff
@@ -0,0 +1,518 @@
+.. _worker-selection:
+
+Selection
+=========
+
+Lightly allows you to specify the subset to be selected based on several objectives.
+
+E.g. you can specify that the images in the subset should be visually diverse, be images the model struggles with (Active Learning),
```

Comment:
Either don't use capital letters for active learning or make it a link

Reply:
I switched to lowercase.